script.js

```diff
@@ -30,3 +30,149 @@ contactLink.addEventListener('click', () => {
+    linkSource: '',
+  },
+  {
+    name: 'Multi-Post Stories <br /> Gain + Glory',
+    description: 'lorem ipsum asdasasasdd  asdasdasd asd d',
+    featuredImage: '',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    liveVersion: '',
+    linkSource: '',
+  },
+  {
+    name: 'Multi-Post Stories <br /> Gain + Glory',
+    description:
+      'lorem  fkwifjijsfvi asfvicasjvasf asdasasasdd  asdasdasd asd d',
+    featuredImage: '',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    liveVersion: '',
+    linkSource: '',
+  },
+  {
+    name: 'Multi-Post Stories <br /> Gain + Glory',
+    description:
+      'lorem asdkfkasndfasdf asdfasd f asdf asdasasasdd  asdasdasd asd d',
+    featuredImage: '',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    liveVersion: '',
+    linkSource: '',
+  },
+  {
+    name: 'Multi-Post Stories <br /> Gain + Glory',
+    description:
+      'lorem adfknkadnfasdf df dasfdfipsum asdasasasdd  asdasdasd asd d',
+    featuredImage: '',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    liveVersion: '',
+    linkSource: '',
+  },
+  {
+    name: 'Multi-Post Stories <br /> Gain + Glory',
+    description:
+      'lorem ipasdjfnkasdf asdf asdf sum asdasasasdd  asdasdasd asd d',
+    featuredImage: '',
+    technologies: ['Ruby on rails', 'css', 'Javascript', 'html'],
+    liveVersion: '',
+    linkSource: '',
+  },
+];
+
+const cardsPortfolio = document.querySelector('.cards-portfolio');
+let portfolioSection = '';
+portfolioObj.forEach((item, index) => {
+  let technologies = '';
+  item.technologies.forEach((tech) => {
+    technologies += `<li>
+    <a href='#'
+      ><button class='btn-hover btn-active btn-pressed'>
+        ${tech}
+      </button></a
+    >
+  </li>`;
+  });
+
+  const newItem = `
+  <div class='project'>
+    <div class='image-back'></div>
+    <div class='project-footer'>
+      <div class='project-title-back'>
+        <h4>${item.name}</h4>
+      </div>
+      <ul class='project-buttons-back'>
+        ${technologies}
+      </ul>
+      <button id='${index}' class='works-button btn-hover btn-active btn-pressed'>
+        See Project
+      </button>
+    </div>
+  </div>
+  `;
+  portfolioSection += newItem;
+});
+
+cardsPortfolio.innerHTML += portfolioSection;
+
+const worksButtons = document.querySelectorAll('.works-button');
+const popupContainer = document.querySelector('.popup-container');
+const popupMenu = document.querySelector('.popup-menu');
+
+worksButtons.forEach((button) => {
+  button.addEventListener('click', (e) => {
+    const portfolioItem = portfolioObj[Number(e.target.id)];
+
+    let technologies = '';
+    portfolioItem.technologies.forEach((tech) => {
+      technologies += `<li>
+      <button class='btn-hover btn-active btn-pressed'>
+        ${tech}
+      </button>
+    </li>`;
+    });
+
+    popupMenu.innerHTML = `
+      <i class='fa fa-close' id='close-popup'></i>
```

Comment:
- [ ] Even at a screen size of 410px, I couldn't see the close button. The close button was only visible around 413px. The minimum mobile screen size should be 375px.

![Capture](https://user-images.githubusercontent.com/63087491/215883375-61a1d02a-c85e-4f5d-8515-bd0b2293f311.PNG)

- [ ] Once more, kindly make sure the pop up follows the proposed design in Figma(font, images, text, margins). The font size for the project **name** both mobile and desktop is not the same as in Figma. Please do some adjustments. The button size for `See live` and `see source` for the mobile menu are not the same as in the original design. Kindly compare and do the adjustments.